snapcraft/plugins/autotools.py

```diff
@@ -32,15 +32,15 @@
       (list of strings)
       configure flags to pass to the build such as those shown by running
       './configure --help'
-    - install-via:
+    - (deprecated) install-via:
```

Comment:
Hmm. This doesn't look deprecated here, it looks completely removed :stuck_out_tongue: . This exists for the (surprisingly abundant) projects that don't respect DESTDIR; we need to make sure we continue to have a solution for them.